docs/contributing/code/patches.rst

```diff
@@ -292,13 +316,20 @@ Rework your Patch
 ~~~~~~~~~~~~~~~~~
 
 Based on the feedback on the pull request, you might need to rework your
-patch. Before re-submitting the patch, rebase with ``upstream/master``, don't merge; and force the push to the origin:
+patch. Before re-submitting the patch, rebase with your base branch (``master`` or ``1.0``), don't merge; and force the push to the origin:
 
 .. code-block:: bash
 
     $ git rebase -f upstream/master
     $ git push --force origin BRANCH_NAME
 
+or
+
+.. code-block:: bash
+
+    $ git rebase -f upstream/1.0
+    $ git push --force origin BRANCH_NAME
```

Comment:
`--force` -> `--force-with-lease`

We should promote best practices everywhere :)

Reply:
Same above